modules/doobie/src/main/scala/doobiemapping.scala

```diff
@@ -197,42 +197,45 @@ object DoobieMapping {
       val preds = predicates.map {
         case (tpe, FieldEquals(fieldName, value)) =>
           val col = columnOfField(tpe, fieldName)
-          Fragment.const(s"${col.toSql} = ") ++ fr"$value"
+          Fragment.const(s"${col.toSql} =") ++ fr0"$value"
 
         case (tpe, FieldLike(fieldName, pattern, caseInsensitive)) =>
           val col = columnOfField(tpe, fieldName)
           val op = if(caseInsensitive) "ILIKE" else "LIKE"
-          Fragment.const(s"${col.toSql} $op ") ++ fr"$pattern"
+          Fragment.const(s"${col.toSql} $op") ++ fr0"$pattern"
 
         case (tpe, AttrEquals(keyName, value)) =>
           val col = columnOfKey(tpe, keyName)
-          Fragment.const(s"${col.toSql} = ") ++ fr"$value"
+          Fragment.const(s"${col.toSql} =") ++ fr0"$value"
 
         case (tpe, AttrLike(keyName, pattern, caseInsensitive)) =>
           val col = columnOfField(tpe, keyName)
           val op = if(caseInsensitive) "ILIKE" else "LIKE"
-          Fragment.const(s"${col.toSql} $op ") ++ fr"$pattern"
+          Fragment.const(s"${col.toSql} $op") ++ fr0"$pattern"
 
         case (tpe, FieldContains(path, value)) =>
           val tpe1 = tpe.path(path.init).underlyingObject
           val col = columnOfField(tpe1, path.last)
-          Fragment.const(s"${col.toSql} = ") ++ fr"$value"
+          Fragment.const(s"${col.toSql} =") ++ fr0"$value"
 
         case (tpe, AttrContains(path, value)) =>
           val tpe1 = tpe.path(path.init).underlyingObject
           val col = columnOfKey(tpe1, path.last)
-          Fragment.const(s"${col.toSql} = ") ++ fr"$value"
+          Fragment.const(s"${col.toSql} =") ++ fr0"$value"
 
         case _ => Fragment.empty
       }
 
-      val where = Fragments.whereAnd(preds ++ joins.map(join => Fragment.const(join.toSql)): _*)
+      // we shouldn't need to do this because `tables` should really just be the root table
+      val tablesʹ = tables.filterNot(joins.map(_.child.table).toSet)
```

Comment:
I think `MappedQuery` should have `table: String` rather than `tables: List[String]` because any other tables we introduce will come from `Join`s ... ?